### ISSUE TITLE
[GALSNV] SuperNova - Endless loop after selecting your opponent -- new connections get hung

### ISSUE THREAD

**Issue by tuday2** [mbbsemu/MBBSEmu#336]
**Module Information**
* Identifier: GALSNV
* Module Name: SuperNova
* Version: 1.0

**Describe the bug**
When playing a game, sometimes you enter an endless loop after selecting your opponent -- new connections get hung

**To Reproduce**
1. Add module to moduleConfig.json
2. Load MBBSEmu
3. Enter module
4. (P) Play game
5. Select "00" for computer, then type a single character and hit enter
6. See error

**Expected behavior**
Able to play game

**Screenshots**
```
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8829 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.toupper Converted a to A
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.prfmsg Added 179 bytes to the buffer from message number 64
2020-12-20 19:21:43.8974 Info MBBSEmu.HostProcess.ExportedModules.Majorbbs.clrprf Reset Output Buffer
```

**Software Information:**
- OS: Windows Server 2019 Standard
- Telnet Client: NetRunner
- MBBSEmu Build: 122020-12

**Comment by enusbaum** [comment]
When I'm running it, I get a missing Ordinal error. I'm wondering if the fixed `printf` formatting that was just merged to master fixed the loop and now we're getting a missing ordinal `MAJORBBS.128` (CNCCIG)

**Comment by enusbaum** [comment]
![image](https://user-images.githubusercontent.com/1139047/102839336-4b43f180-43ce-11eb-81ff-bd62fbc24980.png)

**Comment by enusbaum** [comment]
@tuday2 Was this fixed with #338 ?

**Comment by tuday2** [comment]
Closed by #338
